function handleError(error) {
  console.error(error);
}

async function foo() {
  await bar();
}
